### PR TITLE
[Google Threat Intelligence] IoC Stream Feed

### PR DESCRIPTION
**Pull request by pabloperezj**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by pabloperezj** [comment]
Here is the demo video! :)
https://github.com/user-attachments/assets/c0cce53c-275a-4e62-9a43-39cd53df5879

File: Packs/GoogleThreatIntelligence/Integrations/IoCStream/IoCStream_description.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+### Authorization:
+Your API key can be found in your Google Threat Intelligence account user menu.
+Your API key carries all your privileges, so keep it secure and don't share it with anyone.
+
+### Cortex XSOAR version 6.0.0 and below:
+Fill anything in the "Username" fill box. Use you API key in the password fill box.
```
Comment (adi88d) [inline Packs/GoogleThreatIntelligence/Integrations/IoCStream/IoCStream_description.md:6]: what do you mean? there is only API Key

**Review by adi88d — CHANGES_REQUESTED** [review]
Hi @pabloperezj,
Thank you for your contribution!
Good work :)
- please remove the **gti-iocstream-reset-fetch-indicators** command, you can click on `Re-fetch indicators from this instance` instead
image

- Note that feed integration should starts with `Feed` - please change the dir name to `FeedIOCStream` and all the files inside `FeedIOCStream_description.md`,`FeedIOCStream_test.py` etc....

Please feel free to reach out to me with any questions - I'm available here or on slack :) Thanks again

File: Packs/GoogleThreatIntelligence/Integrations/FeedIoCStream/FeedIoCStream.py [hunk 2]
```diff
@@ -0,0 +1,459 @@
...
+    indicator_type = _get_indicator_type(item)
+    indicator_id = _get_indicator_id(item)
+
+    attributes = item.get('attributes', {})
+    context_attributes = item.get('context_attributes', {})
+
+    detection_ratio = DetectionRatio(attributes.get('last_analysis_stats', {}))
+
+    indicator_obj = {
+        'type': indicator_type,
```
Comment (adi88d) [inline Packs/GoogleThreatIntelligence/Integrations/FeedIoCStream/FeedIoCStream.py:259]: why you didnt added **score** value to the **indicator_obj**?
please see this articles:
https://xsoar.pan.dev/docs/integrations/dbot#score-types
https://xsoar.pan.dev/docs/integrations/dbot#score-types

**Comment by adi88d** [comment]
@pabloperezj
The code looks good!

We're ready for a demo. Please check [this page](https://xsoar.pan.dev/docs/contributing/demo-prep), and let me know when you're available for one over DFIR. 
Feel free also to send me a recording of a demo.

**Comment by Ni-Knight** [comment]
Looks good, too small notes:
1. please normalize the GTI verdict and input it as an indicator verdict as well (@adi88d  commented about it as well). 
2. If you can please also add relationships to malware names, and threat actors (Title Capitalized) that would be great.

File: Packs/GoogleThreatIntelligence/Integrations/FeedIoCStream/FeedIoCStream.py (hunk 2 above)
Comment (pabloperezj) [reply Packs/GoogleThreatIntelligence/Integrations/FeedIoCStream/FeedIoCStream.py:269]: Done!

**Comment by pabloperezj** [comment]
> Looks good, too small notes:
> 
> 1. please normalize the GTI verdict and input it as an indicator verdict as well (@adi88d  commented about it as well).
> 2. If you can please also add relationships to malware names, and threat actors (Title Capitalized) that would be great.

1. Done!
2. Sure :)

**Comment by Ni-Knight** [comment]
@pabloperezj I've added (3) after you answered, sorry I didn't notice it the first time around.

**Comment by pabloperezj** [comment]
> @pabloperezj I've added (3) after you answered, sorry I didn't notice it the first time around.

I will need the new indicator fields in both Packs, what should I do about `IndicatorFields` folder?

**Comment by Ni-Knight** [comment]
I think the best solution is having the indicator fields in the Pack thats already in the market. In the feed make the Google Threat Intel pack mandatory dependency which will mean the pack is installed with it.

**Comment by Ni-Knight** [comment]
Okay, never mind about (3), leave it as is.

**Comment by adi88d** [comment]
@pabloperezj are you ready for the demo? let me know when you're available for one over DFIR.
thanks

**Comment by pabloperezj** [comment]
> @pabloperezj are you ready for the demo? let me know when you're available for one over DFIR. thanks

Isn't the demo video sufficient as in previous occasions? I can upload a new demo video with the latest changes. If a demo call is needed, I am available next Monday.

**Comment by adi88d** [comment]
@pabloperezj Great demo today!
- please change the default Source Reliability to `C - Fairly reliable`
- change the readable output for the `gti-iocstream-get-indicators` command to show the indicators id and value

**Review by adi88d — APPROVED** [review]
Great!

**Pull request merged**